cli/pcluster/config/validators.py

```diff
@@ -420,6 +422,24 @@ def _get_pcluster_user_policy(partition, region, account_id):
     ]
 
 
+def ec2_instance_type_validator(param_key, param_value, pcluster_config):
+    errors = []
+    warnings = []
+
+    try:
+        if param_value not in get_supported_instance_types():
+            errors.append(
+                "The instance type '{0}' used for the '{1}' parameter is not supported by AWS ParallelCluster.".format(
+                    param_value, param_key
+                )
+            )
+
+    except ClientError as e:
```

Comment:
get_supported_instance_types is exiting with error on ClientError so this code can never be reached.

Reply:
Good catch! resolved